rclcpp/src/rclcpp/logger.cpp

```diff
@@ -80,10 +81,12 @@ Logger::get_child(const std::string & suffix)
   {
     std::lock_guard<std::recursive_mutex> guard(*logging_mutex);
     rcl_ret = rcl_logging_rosout_add_sublogger((*name_).c_str(), suffix.c_str());
-    if (RCL_RET_OK != rcl_ret) {
+    if (RCL_RET_NOT_FOUND == rcl_ret) {
+      rcl_reset_error();
```

Comment:
how about adding warning that tells user there will be no rosout since parent logger is not available aligned with https://github.com/ros2/rclpy/pull/1196#discussion_r1388268376

Reply:
I think should not print warning message if `rosout is diabled explicitly` by users.